src/components/IngredientCard/IngredientCard.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Themes from 'assets/json/themes.json';
+
+import './IngredientCard.scss';
+
+const IngredientCard = ({
+  name,
+  image,
+  theme,
+  children,
+}) => (
+  <div className={`ingredient-card ${Themes[theme] || Themes.default}`}>
+    <div className="ingredient-header">
+      <img
+        className="ingredient-image"
+        alt={name}
+        src={image}
+      />
+      <div className="ingredient-name">
+        {name}
```

Comment:
Why the new line here? We can add it on the same line

Reply:
New line is more cleaner.